frontend/src/components/pages/RegistrantExperience/index.tsx

```diff
@@ -60,6 +76,17 @@ const RegistrantExperiencePage = (): React.ReactElement => {
     }
   };
 
+  const stepNavigation = (steps = 1) => {
```

Comment:
When do we need to skip more than one?

Reply:
There's no case where we skip more than one rn. But I also implemented like this to handle going back (-1 steps)

Reply:
Hmm the default doesn't help much with clarity -- can we at least include a type?